android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -151,6 +151,9 @@
 	public static final int REDIRECTS = 5;
 
 	private TiFile responseFile;
+	private boolean hasAuthentication = false;
+	private String username;
```

Comment:
Set line 106 to:
```Java
private HTTPClientProxy proxy;
```

and line 402 to:
```Java
this.proxy = (HTTPClientProxy) proxy;
```

so you can use `proxy.getUsername()` and `proxy.getPassword()` instead?